test/e2e/storage/drivers/in_tree.go

```diff
@@ -1241,6 +1241,8 @@ func (h *gcePdDriver) GetClaimSize() string {
 }
 
 func (g *gcePdDriver) CreateDriver() {
+	// Reset config. It might have been modified by a previous CreateVolume call.
+	g.driverInfo.Config.ClientNodeSelector = nil
```

Comment:
Instead of clearing out just this specific field, can we always reset the entire config before every test case across all drivers?  I don't see a reason why we need to persist any config between tests.

Reply:
Yes, that's also possible.

Reply:
PR updated.

Reply:
Sorry I meant can we do this at the framework level so that every plugin doesn't have to do it?

Reply:
That seems a nice idea. Init***Driver returns Config on the same position and we could reset it on the same way at https://github.com/kubernetes/kubernetes/blob/master/test/e2e/storage/in_tree_volumes.go#L76

Reply:
That also works. I think its a bit inconsistent with how the interface is currently defined, but hopefully we'll change that anyway soon.

I've updated the PR.